code/modules/client/preference_setup/general/03_body.dm

```diff
@@ -449,7 +449,7 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 			pref.r_hair = 0//hex2num(copytext(new_hair, 2, 4))
 			pref.g_hair = 0//hex2num(copytext(new_hair, 4, 6))
 			pref.b_hair = 0//hex2num(copytext(new_hair, 6, 8))
-			pref.s_tone = 0
+			pref.s_tone = -75

```

Comment:
Have you tested that this doesn't mess with non-human colors?

Reply:
Played fine with my Skrell.

Reply:
Even on a new slot changing from human to skrell etc?

Reply:
Correct. I'll test it with other species some time today.

Reply:
Other species use the same system, tis fine if it worked on the Squid